store/sqlstore/upgrade.go

```diff
@@ -168,14 +168,17 @@ func UpgradeDatabaseToVersion33(sqlStore SqlStore) {
 			if err != nil {
 				themeMigrationFailed(err)
 			}
+			defer transaction.Rollback()
```

Comment:
Given the `themeMigrationFailed` nuances here, and the fact that this is a /very/ old upgrade path, I'd propose not modifying this code at all since we're unlikely to worry about the original error report at this level and there's a non-zero risk to touching really old upgrade code.

Reply:
Sure thing. I can revert the changes for this file.